docs/dtc-BaseDTCRetriever.js

```
@@ -0,0 +1,2485 @@
+            );
+
+            // Update our response variables with the new result
+            const { responseString: newResponse, rawResponse: newRawResponse } =
+              retryResult;
+
+            if (newResponse && !this.isErrorResponse(newResponse)) {
+              log(
+                'debug',
+                'Successfully got response after flow control configuration',
+              );
+              testResponse = newResponse;
```

Biome (1.9.4)

[error] Can't assign testResponse because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)